geomapshark/settings.py

```diff
@@ -575,3 +567,26 @@
         "django.contrib.auth.backends.RemoteUserBackend",
         "django.contrib.auth.backends.ModelBackend",
     ]
+
+CKEDITOR_CONFIGS = {
+    "default": {
+        # TODO: customize style and format dropdowns
+        "toolbar": "custom",
+        "toolbar_custom": [
+            ["Undo", "Redo"],
+            ["Image", "Table", "HorizontalRule", "SpecialChar"],
+            ["Bold", "Italic", "Strike", "-", "RemoveFormat"],
+            [
+                "NumberedList",
+                "BulletedList",
+                "-",
+                "Outdent",
+                "Indent",
+                "-",
+                "Blockquote",
+            ],
+            ["Styles", "Format"],
+            ["Source"],
+        ],
+    },
+}
```

Comment:
> Define the scope of the CKEDITOR

You mean name the config `reports` (or similar) instead of `default` ? IMO this could be done the day CKEDITOR is needed elsewhere, because when not using default, one must manually specify the config

> Remove functions that are not intended to be used

Here it's really a decision on your side. IMO the least possibilities is usually the better. Maybe you could leave it with many features to start with, then after a bit of usage review what's actually been used. The content of the fields are HTML anyway, so theoretically it should be backwards compatible, even if you remove used features.